src/lib/components/DeckGLMap/layers/wells/wellsLayer.js

```diff
@@ -0,0 +1,31 @@
+import { CompositeLayer } from "@deck.gl/core";
+import { GeoJsonLayer } from "@deck.gl/layers";
+
+export default class WellsLayer extends CompositeLayer {
```

Comment:
Doesn't need to be in the same review, it could be ported later, but ideally all new layers are in typescript.